src/external/atomsbase.jl

```diff
@@ -19,15 +19,23 @@ function parse_system(system::AbstractSystem{D}) where {D}
     atoms = map(system) do atom
         pseudo = get(atom, :pseudopotential, "")
         if isempty(pseudo)
-            ElementCoulomb(atomic_number(atom))
+            ElementCoulomb(atomic_number(atom); mass=atomic_mass(atom))
         else
-            get!(cached_pspelements, pseudo) do
+            key = pseudo * string(get(atom, :atomic_mass, ""))
+            get!(cached_pspelements, key) do
                 kwargs = get(atom, :pseudopotential_kwargs, ())
-                ElementPsp(atomic_number(atom); psp=load_psp(pseudo; kwargs...))
+                ElementPsp(atomic_number(atom); psp=load_psp(pseudo; kwargs...),
+                           mass=atomic_mass(atom))
             end
         end
     end
 
+    dftk_atomic_masses = getfield.(element.(getfield.(atoms, :symbol)), :atomic_mass)
+    if (atomic_mass(system) != dftk_atomic_masses)
+        @warn "Some atomic masses differ from the default ones; they may come from \
+               rounding error from the external parser."
+    end
+
```

Comment:
Don't need this any more.

Reply:
I don't think that's worth it. It will only be noise as this is far too sensitive.